src/main/java/com/icthh/xm/tmf/ms/resource/config/CloudDatabaseConfiguration.java

```diff
@@ -10,13 +9,15 @@
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
 
+import javax.sql.DataSource;
```

Comment:
Unnecessary changes, according checkstyle imports orders must be the same as in old version